frontend/src/metabase-lib/metadata/Table.ts

```diff
@@ -65,7 +65,7 @@ class Table {
   }
 
   newQuestion() {
-    return this.question().setDefaultQuery().setDefaultDisplay();
```

Comment:
After removing GA-specific code, it looked like `setDefaultQuery()` didn't do anything anymore, so I removed it. I could be wrong 🤔